configs/tdh-test.yml

```diff
@@ -0,0 +1,325 @@
+estimation_buffer: 0.5
+estimation_growth_factor: 0.01  # buffer increases by 1% per month
+storage_buffer: 0.33  # keep max storage at 75% of disk
+storage_display_unit: TB
+
+summary_dates:
+  - '2020-12'
+
+vm_os_storage_gb: 50
+vm_os_storage_group: 'VM_os'
+
+usage:
+  users:
+    model: 'date_range_value'
+    ranges:
+      - ['20180101', '20201201', 1132]
+
+  # LS User as a percent of total User
+  ls_users:
+    model: 'derived_factor'
+    dependant_field: 'users'
+    factor: 0.05
+  # Number of formaplayer user from X date to Y
+  formplayer_users:
+    model: 'date_range_value'
+    ranges:
+      - ['20180901', '20201201', 1000]
+
+  #Number of forms per user per month
+  forms_monthly:
+    model: 'derived_factor'
+    dependant_field: 'users'
+    factor: 0  # https://docs.google.com/spreadsheets/d/1cvgcihfE0oOrkAPW6RlmSLzGcsaD-3nDAOKVCYjyAeE/edit#gid=1011017089
+  forms_daily:
+    model: 'derived_factor'
+    dependant_field: 'forms_monthly'
+    factor: 0
+  # Total Forms ever created
+  forms_total:
+    model: 'cumulative'
+    dependant_field: 'forms_monthly'
+    start_with: 16000000   # from ES form index (total docs)
+
+  #Number of cases per user per month
+  cases_total:
+    model: 'baseline_with_growth'
+    dependant_field: 'users'
+    baseline: 0  # inflated number by MWCD. Actual probably closer to 1000
+    monthly_growth: 0  # guess (4 cases for every new person case)
```

Comment:
Can you remove these old comments/

right now this is saying htat there are no cases ever added. Can you update baseilne with the number of cases per user and growht with number of new cases?

Reply:
Because this is a snapshot for 1 month, is it necessary to add the numbers for growth?

Reply:
I believe that this is a snapshot for one month after the server has been running for X months as defined by https://github.com/dimagi/commcare_resource_model/pull/22/files#diff-3afef5cb6ba552612488865189106b6eR16

so ` - ['20180101', '20201201', 1132]` means that 1132 users have been submitting forms and cases according to these baseline & monthly growht numbers from 2018-01-01 through 2020-12-01 and the final numbers are what resources you would need at the end of 2020